src/veupath/redmine/client/rnaseq.py

```diff
@@ -13,10 +13,13 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+"""Redmine issue object for RNA-Seq datatype."""
 
 import re
-from unidecode import unidecode
 from typing import Any, Dict, List
+
+from unidecode import unidecode
```

Comment:
I may be wrong, but this is not used anywhere, is it?
```suggestion
```

Reply:
It's used in the `_normalize_name` method